### PR TITLE
Cloud bigtable

### PR DESCRIPTION
**Pull request by hildrum**
Add the necessary scopes to the benchmark configuration for the `cloud_bigtable_ycsb` benchmark; for this to work, also needed to move the CheckPrerequisites step to after the flags from benchmark configs have been applied.  

Also, have the cloud_bigtable_ycsb pick up the hbase version from the hbase package.

### CODE REVIEW

**Comment by hildrum** [comment]
@cmccoy PTAL.

File: perfkitbenchmarker/linux_benchmarks/cloud_bigtable_ycsb_benchmark.py [hunk 1]
```diff
@@ -72,6 +79,10 @@
     default:
       vm_spec: *default_single_core
       vm_count: null
+  flags:
+    gcloud_scopes: >
+      https://www.googleapis.com/auth/bigtable.admin
+      https://www.googleapis.com/auth/bigtable.data
```
Comment (cmccoy) [inline perfkitbenchmarker/linux_benchmarks/cloud_bigtable_ycsb_benchmark.py]: Oh, this is much better.

**Comment by cmccoy** [comment]
LGTM. @skschneider - any concerns about the prereqs change?

File: perfkitbenchmarker/pkb.py [hunk 2]
```diff
@@ -355,19 +355,19 @@ def RunBenchmark(benchmark, collector, sequence_number, total_benchmarks,
...
-      detailed_timer = timing_util.IntervalTimer()
+      # Optional prerequisite checking.
+      check_prereqs = getattr(benchmark, 'CheckPrerequisites', None)
+      if check_prereqs:
+        try:
+          check_prereqs()
+        except:
+          logging.exception('Prerequisite check failed for %s', benchmark_name)
+          raise
+        end_to_end_timer = timing_util.IntervalTimer()
```
Comment (ehankland) [inline perfkitbenchmarker/pkb.py]: Looks like these two lines might have accidentally gotten indented

**Comment by skschneider** [comment]
connormccoy@, RE: Moving CheckPrerequisites, I'm in favor of it. It would solve one half of #907. There's just the concern that ehankland@ pointed out.

File: perfkitbenchmarker/pkb.py (hunk 2 above)
Comment (hildrum) [inline perfkitbenchmarker/pkb.py]: Good catch.

**Comment by hildrum** [comment]
PTAL.  Also, add to eliminate the newline at the end of the benchmark config.

**Pull request merged**